code/unclassified/src/average/average.c

```diff
@@ -1,9 +1,12 @@
 // Part of Cosmos by OpenGenus Foundation
 // Code written by Adeen Shukla (adeen-s)
-#include<stdio.h>
+#include <stdio.h>
 
-int main() {
-	int n, sum = 0, tmp = 0, i;
+int 
+main() 
+{
+	int n, tmp = 0, i;
```

Comment:
Please fix the indentation.

Reply:
The indentation is with me correct. I see no problem to the previous one.